rllib/evaluation/worker_set.py

```diff
@@ -689,20 +687,18 @@ def foreach_worker_with_id(
         self,
         func: Callable[[int, RolloutWorker], T],
         *,
-        local_worker=True,
-        # TODO(jungong) : switch to True once Algorithm is migrated.
-        healthy_only=False,
+        local_worker: bool = True,
+        healthy_only: bool = True,
```

Comment:
Just out of curiosity: Why can we switch to True and what migration was Jun's comment directed at?